### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by imoralesc**
ADD Helpdesk_Nacho

### CODE REVIEW

File: helpdesk_factorlibre/models/helpdesk_ticket.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    assigned_date = fields.Datetime(
+        string='Assigned Date',
+        compute='_compute_assigned_date',
+        store=True,
+    )
+    
+    closed_date = fields.Datetime(string='Closed Date')
+    
+    priority = fields.Selection(selection=[
+        ('0', _('Baja')),
```
Comment (DarioLodeiros) [inline helpdesk_factorlibre/models/helpdesk_ticket.py]: avoid using Spanish in the code

File: helpdesk_factorlibre/models/inherit_res_partner.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    
+    count_tickets = fields.Integer(
+        string='Number of tickets',
+        compute='_compute_count_tickets'
+    )
+    
+    @api.multi
+    def _compute_count_tickets(self):
+        for record in self:
+            #Codigo mas eficiente: record.count_tickets = len(record.helpdesk_ticket_ids)
```
Comment (DarioLodeiros) [inline helpdesk_factorlibre/models/inherit_res_partner.py]: this comment for example...
#REVIEW: use "record.count_tickets = len(record.helpdesk_ticket_ids)" to efficient code

**Review by DarioLodeiros — CHANGES_REQUESTED** [review]

File: helpdesk_factorlibre/models/inherit_res_partner.py (hunk 2 above)
Comment (imoralesc) [reply helpdesk_factorlibre/models/inherit_res_partner.py]: Fix in 31c0c3d9567a5b7680d44351437cb1eacb729ef6

File: helpdesk_factorlibre/models/helpdesk_ticket.py (hunk 1 above)
Comment (imoralesc) [reply helpdesk_factorlibre/models/helpdesk_ticket.py]: Fix in 31c0c3d9567a5b7680d44351437cb1eacb729ef6

**Pull request merged**